### DISCUSSION TITLE
cannot delete user from authentication/users table

### DISCUSSION THREAD

**Discussion by ahmedabdelhafez** [supabase/supabase#5615] [Questions] [answered]
hello Supabase team,
i cannot delete accounts from users table, i receive this message
![image](https://user-images.githubusercontent.com/25756993/155231128-098744cf-d679-4946-9817-fe6dd30bca8d.png)

can any one advice or help, thanks.

**Comment by burggraf** [comment, marked as answer]
It looks like you can't delete this user because the user has storage objects attached to their account.  The best way to handle this is to delete the related storage items first then delete the user, or you could try using a `delete cascade` but I'm not sure if that'll leave orphaned objects, so I'd try deleting them first.

**Comment by awalias** [comment]
whilst we figure out a permanent solution there is a workaround here: https://github.com/supabase/storage-api/issues/65#issuecomment-899946126
